<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LUMIS实时能谱</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_16">
    <item>
     <widget class="DockArea" name="dockArea" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>功能</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>基线设置</string>
     </property>
     <addaction name="action_baseline_importFromData"/>
     <addaction name="action_baseline_changeBaseline"/>
     <addaction name="action_baseline_measureBaseline"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="action_configuration"/>
    <addaction name="action_dataPlayBack"/>
    <addaction name="action_offlineDataShow"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_toolBox">
   <property name="maximumSize">
    <size>
     <width>450</width>
     <height>524287</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>工具箱</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="page_linkBehave">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>314</width>
          <height>348</height>
         </rect>
        </property>
        <attribute name="label">
         <string>连接操作</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_8">
           <property name="title">
            <string>初始化</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QPushButton" name="pushButton_clockSynchronization">
              <property name="text">
               <string>时钟同步</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_reset">
              <property name="text">
               <string>复位</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>选择配置模式</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QComboBox" name="comboBox_configFile">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_config">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>配置</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>手动发送命令</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="lineEdit_sendCommand"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_sendCommand">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>28</height>
               </size>
              </property>
              <property name="text">
               <string>发送</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLCDNumber" name="lcdNumber_h_m">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="digitCount">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber_s_ms"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_dataReceive">
             <property name="text">
              <string>开始接收数据</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>定时器</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QSpinBox" name="spinBox_timer_hour">
              <property name="suffix">
               <string>小时</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_timer_minute">
              <property name="suffix">
               <string>分钟</string>
              </property>
              <property name="maximum">
               <number>59</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_timer">
              <property name="text">
               <string>开启定时器</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>88</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>279</width>
          <height>451</height>
         </rect>
        </property>
        <attribute name="label">
         <string>能谱图像</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>能谱显示</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>所在层：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_singal_tier"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>所选通道：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_singal_channel"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>基线设置：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_baseLine">
                <property name="minimum">
                 <number>0</number>
                </property>
                <property name="maximum">
                 <number>4095</number>
                </property>
                <property name="value">
                 <number>415</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_singal_addPlot">
                <property name="text">
                 <string>添加图像</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>符合能谱</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>所在层：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_coincidence_tier"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>显示通道：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_coincidence_showChannel"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>符合通道：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_coincidence_coinChannel"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>基线设置：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_coincidence_baseline">
                <property name="maximum">
                 <number>4025</number>
                </property>
                <property name="value">
                 <number>415</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_coincidence_addPlot">
                <property name="text">
                 <string>添加图像</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string>单层各通道能谱</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>选择层：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_oneLayerChannel_tier">
              <item>
               <property name="text">
                <string>0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>7</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_oneLayerChannel_addPlot">
              <property name="text">
               <string>添加图像</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>300</width>
          <height>294</height>
         </rect>
        </property>
        <attribute name="label">
         <string>检测工具</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>触发事件</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_15">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>触发情况：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_triggerCondition_addPlot">
                <property name="text">
                 <string>添加图像</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_18">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>XY触发情况：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_XYTriggerCondition_addPlot">
                <property name="text">
                 <string>添加图像</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_9">
           <property name="title">
            <string>3D成像</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_19">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>3D图像：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_threeD_addPlot">
                <property name="text">
                 <string>添加图像</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>195</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_alarm">
        <attribute name="label">
         <string>警报功能</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QGroupBox" name="groupBox_10">
           <property name="title">
            <string>报警设置</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QCheckBox" name="checkBox_alarm_messageBox">
              <property name="text">
               <string>开启弹窗警告</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_messageQueue">
   <property name="windowTitle">
    <string>消息队列</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextBrowser" name="textBrowser_messageQueue"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_baseline_importFromData">
   <property name="text">
    <string>从数据导入基线</string>
   </property>
  </action>
  <action name="action_baseline_changeBaseline">
   <property name="text">
    <string>修改基线</string>
   </property>
  </action>
  <action name="action_configuration">
   <property name="text">
    <string>仪器配置参数</string>
   </property>
  </action>
  <action name="action_baseline_measureBaseline">
   <property name="text">
    <string>测量基线数据</string>
   </property>
  </action>
  <action name="action_dataPlayBack">
   <property name="text">
    <string>数据回放</string>
   </property>
  </action>
  <action name="action_offlineDataShow">
   <property name="text">
    <string>离线数据显示</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DockArea</class>
   <extends>QWidget</extends>
   <header>pyqtgraph.dockarea</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
